kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -3289,37 +3293,87 @@ def _dump_ha_network_group_debug(self, stage, down_monitors=None, network_group_
         cmd = ReportHaNetworkGroupStatusCmd()
         cmd.hostUuid = host_uuid
         cmd.networkGroupStatus = network_group_status
-
         try:
             http.json_dump_post(url, cmd, {'commandpath': self.REPORT_HA_NETWORK_GROUP_STATUS_PATH})
-            with self.ha_network_group_lock:
-                self.ha_network_group_last_status = dict(network_group_status)
             logger.debug('reported ha network group status for host[%s], status:%s' % (host_uuid, network_group_status))
             self._dump_ha_network_group_debug('report-status', network_group_status=network_group_status)
+            return True
         except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)